web/src/components/customer/listing-details/CommitFeedbackPrompt.tsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {useState, useEffect} from 'react';
+
+import CommitsBadge from 'components/common/CommitsBadge';
+import MobilePrompt from 'components/common/MobilePrompt';
+
+import {ReactComponent as CelebrateSvg} from 'assets/celebrate.svg';
+import {ReactComponent as PaymentSvg} from 'assets/payment.svg';
+
+import {useCommitFeedbackPromptContext} from './contexts/CommitFeedbackPromptContext';
+
+interface PromptProps {
+  isVisible: boolean;
+  onClose: () => void;
+}
+
+const SuccessfulCommitPrompt: React.FC<PromptProps> = ({
+  isVisible,
+  onClose,
+}) => (
+  <MobilePrompt
+    isVisible={isVisible}
+    title="You have committed to the listing!"
+    header={<CelebrateSvg />}
+    buttons={[
+      {
+        name: 'Manage',
+        // TODO: Link to My Commits page onClick instead
+        onClick: onClose,
+      },
+      {
+        name: 'Dismiss',
+        onClick: onClose,
+      },
+    ]}
+  >
+    <CommitsBadge />
+  </MobilePrompt>
+);
+
+const SuccessfulPaymentPrompt: React.FC<PromptProps> = ({
```

Comment:
Payment prompt looks similar to the commit prompt, do we need two?

Reply:
@chshda I did abit of refactoring for this :)